web/annotation_editor_layer_builder.css

```
@@ -88,6 +98,7 @@
   outline: var(--hover-outline);
 }
 
+.annotationEditorLayer .inkEditor.disabled:focus,
```

This line looks *identical* to the one below, did you mean the following?
```suggestion
.annotationEditorLayer .freeTextEditor.disabled:focus,
```